packages/netsuite-adapter/src/client/sdf_parser.ts

```diff
@@ -231,6 +266,31 @@ const transformFolders = (
   )
 }
 
+const transformFoldersWithoutAttributes = (
+  foldersFromAttributes: FolderCustomizationInfo[],
+  filePaths: string[],
+): FolderCustomizationInfo[] => {
+  const foldersFromAttributesPaths = new Set(
+    foldersFromAttributes
+      .map(folder => folder.path.join(FILE_CABINET_PATH_SEPARATOR))
+  )
+  return _(filePaths)
+    .map(path => path.split(FILE_CABINET_PATH_SEPARATOR))
+    .map(path => path.slice(1, -1))
+    // adding all parent folders
+    .flatMap(path => path.map((p, i) => [...path.slice(0, i), p]))
```

Comment:
super nit: I think it's a bit more readable, but as you prefer
```suggestion
    .flatMap(path => path.map((_, i) => path.slice(0, i + 1)))
```